src/viewport.js

```diff
@@ -133,9 +133,17 @@ $.Viewport = function( options ) {
         animationTime:   this.animationTime
     });
 
+    this.degreesSpring = new $.Spring({
+        initial: options.degrees,
+        springStiffness: this.springStiffness,
+        animationTime: this.animationTime
+    });
+    delete options.degrees;
```

Comment:
As much as possible, we need to not break existing apps, but I think it's good to deprecate `degrees`. Let's make a getter and setter for it that spits out a console.warn, but otherwise calls `this.getRotation` and `this.setRotation` respectively. That way old code will still work, but the developer will be informed that they should change it.

Reply:
That makes sense! Should I add the getter and setter where `options` is defined, or add them later via `Object.defineProperty`? Any preference?